app/components/util/HowDoYouFeel.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+
+const responseOptions = ['Awesome!', 'Just Okay', 'Unsatisfied']
+
+export default function HowDoYouFeel(props) {
+    //Isolate props and set initial state
+    const { disabled, className, onDone, ...otherProps } = props
+    const styleClasses = rankingStyleClasses(props)
+    const classes = buttonStyles()
+    const combinedClassName = cx(classes.Howdoyoufeel, className)
+    const textClasses = textStyles()
+    const textClassName = cx(textClasses.textStyle)
```

Comment:
it is not necessary to create textClassName, or call cx() if there is only one class.